api/api.proto

```diff
@@ -767,6 +767,15 @@ service Wallet {
 
   rpc GetBlock (BlockReq) returns (BlockExtention) {
   }
+
+  rpc GetBandwidthPrices (EmptyMessage) returns (PricesResponseMessage) {
+  }
+
+  rpc GetEnergyPrices (EmptyMessage) returns (PricesResponseMessage) {
+  }
+
+  rpc GetMemoFee (EmptyMessage) returns (PricesResponseMessage) {
+  }
```

Comment:
Why WalletSolidity lack GetMemoFee?

Reply:
Maybe the developer forgot at the time.

Reply:
This API is only implemented in HTTP, without solidity and PBFT.